.gitignore

```diff
@@ -1,7 +1,5 @@
 # snax stuff
-*generated*
 *logs*
-*results*
 
```

Comment:
Why do these need to go?

Reply:
the action needs to commit files that are called result, and this was not happening, pr #268 puts it on the benchmark folder specifically so the convenience of a clean staged file list still remains